libraries/AP_GPS/AP_GPS_UBLOX.h

```diff
@@ -38,8 +38,7 @@
  * modules are configured with all ubx binary messages off, which
  * would mean we would never detect it.
  */
-#define UBLOX_SET_BINARY "\265\142\006\001\003\000\001\006\001\022\117$PUBX,41,1,0003,0001,38400,0*26\r\n"
-#define UBLOX_SET_BINARY_RAW_BAUD "\265\142\006\001\003\000\001\006\001\022\117$PUBX,41,1,0003,0001,115200,0*1E\r\n"
+#define UBLOX_SET_BINARY          "\265\142\006\001\003\000\001\006\001\022\117$PUBX,41,1,0023,0001,115200,0*1C\r\n"
```

Comment:
Have you  confirmed that requesting the additional output protocols doesn't break any of the older u-blox hardware? (I'm specifically thinking 5 and 6's as I lack devices to test this against, I can test 7's and 8's here)